piccolo_api/register/endpoints.py

```diff
@@ -158,21 +164,29 @@ def register(
     auth_table: t.Optional[t.Type[BaseUser]] = None,
     redirect_to: t.Union[str, URL] = "/login/",
     template_path: t.Optional[str] = None,
+    user_defaults: t.Optional[t.Dict[str, t.Any]] = None,
 ) -> t.Type[RegisterEndpoint]:
     """
     An endpoint for register user.
 
     :param auth_table:
-        Which table to create the user in. If not specified, it defaults to
+        Which ``Table`` to create the user in. If not specified, it defaults to
         :class:`BaseUser <piccolo.apps.user.tables.BaseUser>`.
     :param redirect_to:
         Where to redirect to after successful registration.
     :param template_path:
         If you want to override the default register HTML template, you can do
         so by specifying the absolute path to a custom template. For example
         ``'/some_directory/register.html'``. Refer to the default template at
-        ``piccolo_api/session_auth/templates/register.html`` as a basis for
+        ``piccolo_api/templates/register.html`` as a basis for
         your custom template.
+    :param user_defaults:
+        These values are assigned to the new user. An example use case is
+        setting ``active = True`` on each new user, so they can immediately
+        login (not recommended for production, as it's better to verify their
+        email address first, but OK for a prototype app)::
```

Comment:
If we use `user_defaults` arg, we need also set a **warning** in docs that is only for prototyping because otherwise there is no point in decouple the `register` from creating the session, and we could immediately log the user in from `register` endpoint (as we did at the beginning).

Reply:
Yeah, it's a good point. They will still have to login to get the session though.